### ISSUE TITLE
[BugFix] [RHEL/6] One instance of "An xccdf:Rule with @id "xccdf_NAMESPACE_rule_security_patches_up_to_date" MUST reference an OVAL component that contains an oval definition of class 'patch'. "

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/scap-security-guide#1090]
When validating current SSG RHEL-6 benchmark with SCAP NIST content test suite on 2016-03-18 (https://jlieskov.fedorapeople.org/2016-03-18-scap-validation-result.html) there's one instance of the following error:
```
169-1 	1(of 1)

ERROR 	SCHEMATRON

scap_org.open-scap_datastream_from_xccdf_ssg-rhel6-xccdf-1.2.xml -
An xccdf:Rule with @id "xccdf_NAMESPACE_rule_security_patches_up_to_date"
MUST reference an OVAL component  that contains an oval definition of class 'patch'. 	

xccdf:Rule xccdf_org.ssgproject.content_rule_security_patches_up_to_date
in ds:data-stream  scap_org.open-scap_datastream_from_xccdf_ssg-rhel6-xccdf-1.2.xml

xccdf:Rule xccdf_org.ssgproject.content_rule_security_patches_up_to_date
in ds:data-stream  scap_org.open-scap_datastream_from_xccdf_ssg-rhel6-xccdf-1.2.xml

Schematron Context:
/*:data-stream-collection[namespace-uri()='http://scap.nist.gov/schema/scap/source/1.2'][1]/*:data-
stream[namespace-uri()='http://scap.nist.gov/schema/scap/source/1.2'][1] 	every $m in 
ds:checklists/ds:component-ref satisfies every $n in xcf:get-component($m)//xccdf:Rule[starts-
with(@id,'xccdf_') and ends-with(@id,'_rule_security_patches_up_to_date')]//xccdf:check-content-
ref[not(xcf:is-external-ref($m/cat:catalog, @href) cast as xsd:boolean)] satisfies exists(xcf:get-
component(xcf:get-component-ref($m/cat:catalog, $n/@href))//oval-def:definition[@class eq 
'patch']) 
```

**Comment by iankko** [comment]
Moving back to the pool so others can have a look too.

**Comment by mpreisler** [comment]
We clearly do satisfy this but with an external remote OVAL. Looks like an issue in SCAPVal, similar to #1088

**Comment by iankko** [comment]
The ```scap-security-guide-nist-testsuite``` Jenkins test job has been modified to whitelist the specific error message for this problem (see next comment for explanation why):
```
ERROR: SCHEMATRON - [ssg-rhel6-ds.xml] scap_org.open-scap_datastream_from_xccdf_ssg-
rhel6-xccdf-1.2.xml - An xccdf:Rule with @id 
"xccdf_NAMESPACE_rule_security_patches_up_to_date" MUST reference an OVAL component 
that contains an oval definition of class 'patch'.
```

**Comment by iankko** [comment]
The reason for this issue (rather than to be a problem in the SSG content itself) has been recognized to be a deficiency in the SCAP NIST content validation itself. As of ```scapval-1.2.1.13.jar``` version it is not able to recognize remotely referenced OVAL check being present in the benchmark, and issuing ```ERROR``` in this case. This behaviour has been confirmed by NIST as not the desired one and as such will change in future ScapVal versions.

Instead of issuing an ```ERROR``` future version will issue a ```WARNING```. Future ScapVal version should be made available by start of April 2016.

For now we have whitelisted this error in the testsuite Jenkins job, and as such no further action is necessary from our end.

Closing as resolved.
